bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -669,7 +671,7 @@ void PreProcessor::sendPreProcessRequestToAllReplicas(const PreProcessRequestMsg
       // sendMsg works asynchronously, so we can launch it sequentially here
       LOG_DEBUG(logger(),
```

Comment:
@nkgeorgiev @yuliasherman does it make sense to bump this log to INFO level?

Reply:
I don't think that we need it. We have handlePreProcessedReqByNonPrimary logs at INFO level, so we can infer that the request was sent and how many replicas received it.

Reply:
@teoparvanov Do you mean LOG_INFO(logger(), KVLOG(reqSeqNum, clientId) << " no consensus reached, request released"); ?
It's a rare situation and we want to know this happened.

Reply:
My suggestion is to bump it to INFO instead of DEBUG, but on the other hand, it might bloat the logs.

Reply:
Why do we need this log to be at the INFO level?

Reply:
Just asking - would it be useful to use this in the pre-processor log parser summary?

Reply:
In this case we could, but I'm not sure it will give a better log parser summary.